src/main/java/pt/iscte/es1/antiSpamFilter/AntiSpamFilterNSGAIIRunner.java

```diff
@@ -0,0 +1,74 @@
+package pt.iscte.es1.antiSpamFilter;
+
+import org.uma.jmetal.algorithm.Algorithm;
+import org.uma.jmetal.algorithm.multiobjective.nsgaii.NSGAIIBuilder;
+import org.uma.jmetal.operator.impl.crossover.SBXCrossover;
+import org.uma.jmetal.operator.impl.mutation.PolynomialMutation;
+import org.uma.jmetal.problem.DoubleProblem;
+import org.uma.jmetal.qualityindicator.impl.hypervolume.PISAHypervolume;
+import org.uma.jmetal.solution.DoubleSolution;
+import org.uma.jmetal.util.experiment.Experiment;
+import org.uma.jmetal.util.experiment.ExperimentBuilder;
+import org.uma.jmetal.util.experiment.component.*;
+import org.uma.jmetal.util.experiment.util.ExperimentAlgorithm;
+import org.uma.jmetal.util.experiment.util.ExperimentProblem;
```

Comment:
Just remove this unnecessary imports:

import pt.iscte.es1.antiSpamFilter.domain.Message;
import pt.iscte.es1.antiSpamFilter.domain.WeightedRule;
import pt.iscte.es1.antiSpamFilter.infrastructure.AntiSpamFileReader;
import pt.iscte.es1.antiSpamFilter.infrastructure.LogParser;
import pt.iscte.es1.antiSpamFilter.infrastructure.RuleParser;
import java.io.FileReader;